bpf/lib/nodeport.h

```diff
@@ -1548,6 +1615,13 @@ static __always_inline int dsr_reply_icmp4(struct __ctx_buff *ctx,
 	if (ctx_store_bytes(ctx, off + sizeof(ip), &icmp,
 			    sizeof(icmp), 0) < 0)
 		goto drop_err;
+	if (ctx_store_bytes(ctx, off + sizeof(ip) + sizeof(icmp),
```

Comment:
As `tmp` has the full and well updated headers I'm wondering whether it would have been better to call `ctx_store_byte()` only one time with it?

Reply:
It would work. I code it this way to improve readability and making intentions clear.